app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -178,6 +178,23 @@ def idv_phone_upload_link_sent(
       )
     end
 
+    # @param [Boolean] success
+    # @param [JSON] link_params
```

Comment:
JSON isn't really a type? this is probably a hash right?
```suggestion
    # @param [Hash] link_params
```

Reply:
I have refactored the code away from using this.  Maybe reload the page? Sorry for making you double review... 😓